### PR TITLE
Update substrate-cli.mdx

### PR DESCRIPTION
**Pull request by PurpleM0ney**
Docker-Compose has been changed to Docker Compose

### CODE REVIEW

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Thank you for your suggestions @PurpleM0ney, requested a few changes!
Please also update the versioned version of the same file, located in `/versioned_docs/version-latest `folder. 

Looking forward to the changes and more PRs coming from you!

**Comment by PurpleM0ney** [comment]
> Thank you for your suggestions @PurpleM0ney, requested a few changes! Please also update the versioned version of the same file, located in `/versioned_docs/version-latest `folder.
> 
> Looking forward to the changes and more PRs coming from you!

Fixed it!

File: docs/protocol/substrate-cli.mdx [hunk 1]
```diff
@@ -147,9 +147,9 @@ We will be downloading two files for your respective operating system.
 
   3. Create the `subspace` directory and the `docker-compose.yaml` file in it. Paste the output from the generator into the file.
   
-  4. Now go to directory with `docker-compose.yaml` and type `docker-compose up -d` to start everything
+  4. Now go to directory with `docker-compose.yaml` and type `docker compose up -d` to start everything
```
Comment (EmilFattakhov) [inline docs/protocol/substrate-cli.mdx:150]: Please also update the versioned version of the same file, which can be found in `/versioned-docs/version-latest/protocol/substrate-cli.mdx`

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Thank you for the update @PurpleM0ney, now the file looks correct to me. 

Please also update the versioned copy of the same file (I've added a link to the file that needs to be updated in a comment), and it should be good to go!

File: docs/protocol/substrate-cli.mdx (hunk 1 above)
Comment (gonner22) [reply docs/protocol/substrate-cli.mdx:150]: Thanks, guys for adding this issue and PR on my behalf. (https://discord.com/channels/864285291518361610/1143624977216712825/1144008595323756685)

Topper

**Comment by gonner22** [comment]
Thanks, guys for adding this issue and PR on my behalf. (https://discord.com/channels/864285291518361610/1143624977216712825/1144008595323756685)

Topper

**Review by EmilFattakhov — APPROVED** [review]
To unblock the community with the updated docker-compose command, I'll push the updates for the versioned file as a part of a separate PR. 

Thank you for your contribution @PurpleM0ney!

**Pull request merged**